shared/src/main/scala/squants/package.scala

```diff
@@ -118,4 +119,13 @@ package object squants {
     def /(that: Time) = Each(bd) / that
     def per(that: Time): Frequency = /(that)
   }
+
+  /**
+    * Provides an implicit conversion from Dimensionless to Double, allowing a Dimensionless value
+    * to be used anywhere a Double (or similar primitive) is required
+    *
+    * @param d Dimensionless
+    * @return
+    */
+  implicit def DimensionlessToDouble(d: Dimensionless): Double = d.toEach
```

Comment:
I think there is a tendency to avoid implicit conversions, preferring something like an implicit class with an explicit `toDouble` method.

On the other hand, I think that won't work on 2.10, right?